app/src/test/java/org/simple/clinic/summary/PatientSummaryEffectHandlerTest.kt

```diff
@@ -179,6 +188,32 @@ class PatientSummaryEffectHandlerTest {
     verifyZeroInteractions(uiActions)
   }
 
+  @Test
+  fun `when the load data for done click effect is received and diagnosis is not recorded, load the data`() {
```

Comment:
We're adding all this boilerplate just to verify that a boolean field is being computed correctly. The computation is essentially a pure function and should be part of business logic, but we are doing this because this is currently in the effect handler.

If we make that earlier change about moving the computation for `diagnosisRequired` into the `MedicalHistory` class, we can remove this test and have very simple tests elsewhere which just validate the computation of the property.